src/services/query.ts

```diff
@@ -21,13 +21,18 @@ export interface ConstructorOptions {
   distribution: Distribution;
   query: string;
   comunica: IActorInitSparqlArgs;
+  timeoutMs: number;
 }
 
 const schemaConstructor = Joi.object({
   logger: Joi.object().required(),
   distribution: Joi.object().required(),
   query: Joi.string().required(),
   comunica: Joi.object().required(),
+  timeoutMs: Joi.number()
+    .integer()
```

Comment:
Should we also define a minimum value with `min()` (e.g. 0 or 1000), to at least prevent nonsense values such as `-1`?